android/media/audio/record_audio/RecordAudio/Activity1.cs

```diff
@@ -28,7 +29,17 @@ protected override void OnCreate (Bundle bundle)
             _start = FindViewById<Button> (Resource.Id.start);
             _stop = FindViewById<Button> (Resource.Id.stop);
                     
-            string path = "/sdcard/test.3gpp";
+			string path = $"{Android.OS.Environment.ExternalStorageDirectory.AbsolutePath}/test.3gpp";
+
+			//Explicit permissions
+			if (Build.VERSION.SdkInt > BuildVersionCodes.M)
```

Comment:
The APK level is set to 24 (Android 7), so this check isn't necessary; it will always pass.